bootstrapper-maven-plugin/src/main/resources/templates/Status.java

```diff
@@ -0,0 +1,7 @@
+package {{groupId}};
+
+import io.javaoperatorsdk.operator.api.ObservedGenerationAwareStatus;
+
+public class {{artifactClassId}}Status extends ObservedGenerationAwareStatus {
```

Comment:
I think we can extend a bit this section, a pretty common pattern is to have a list of conditions.

Reply:
Ahh want to have this as simple as possible. TBH don't think conditions are that common. But might be wrong

Reply:
@csviri quick feedback:
https://github.com/kubernetes/community/blob/4c9ef2d/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties

From my understanding `Conditions` IS the "default way" to expose the Status in a CRD and making it easy in the JOSDK would be great 🙂

Reply:
To my understanding it's not, it was even discussed at some point to remove from k8s. Do you have some reference on this?

Reply:
There is a nice article about the history, it seems that they are going to stay:
https://maelvls.dev/kubernetes-conditions/

But is it hard now, to expose the conditions, how could we help with that? The class representation is already there to use. Maybe we should create an issue to discuss it there?

Reply:
I'm already grateful for the great link :-P thanks Attila!

> how could we help with that?

Maybe exposing utilities to have an easy way to do "Conditions" in the "right" way.